api/requirements.txt

```
@@ -85,3 +85,4 @@ pymysql==1.1.1
 tidb-vector==0.0.9
 google-cloud-aiplatform==1.49.0
 vanna[postgres,mysql,clickhouse,duckdb]==0.5.5
+tencentcloud-sdk-python-hunyuan==3.0.1158
```

```suggestion
tencentcloud-sdk-python-hunyuan~=3.0.1158
```
This dependency package maintains a release cycle of two to three days for a new version.
Maybe we could use the "~=" operator to keep track.